includes/widgets/divider.php

```diff
@@ -437,7 +437,7 @@ protected function register_controls() {
 					'unit' => '%',
 				],
 				'selectors' => [
-					'{{WRAPPER}} .elementor-divider-separator' => 'width: {{SIZE}}{{UNIT}};',
+					'{{WRAPPER}} .elementor-divider-separator' => '--divider-width: {{SIZE}}{{UNIT}};',
```

Comment:
@TzviRabinovitch 

I think this will cause a problem, because:

1.  in the scss the `--container-widget-width` overwrites the  `--divider-width`.
2. I think for BC reasons it is better to keep the original styling. E.g. third party plugins could have based their styling based on this specificy.

My suggestion would be not to make any changes here.

Reply:
Done